README.md

```diff
@@ -176,6 +182,26 @@ An import will also cause dfe-analytics-dataform to correct occasions when there
+
+To use hidden field functionality in ```dfe-analytics-dataform``` effectively, you must first:
+1. Set up IAM roles and policy tags in your GCP project, and attach the policy tags to the correct field in your events table, following the instructions documented [here](https://github.com/DFE-Digital/dfe-analytics/blob/main/docs/google_cloud_bigquery_setup.md).
+2. Configure ```dfe-analytics``` to treat the desired fields as hidden by listing them in your application's ```config/analytics_hidden_pii.yml``` file as described [here](https://github.com/DFE-Digital/dfe-analytics/tree/main?tab=readme-ov-file#4-send-database-events).
+
+You must then configure the following dfe-analytics-dataform parameters as appropriate:
+- ```hiddenPolicyTagLocation``` - string; optional; top level or key level. The location of a policy tag that you wish to attach to hidden fields, for example projects/your-project-name/locations/europe-west2/taxonomies/your-taxonomy-id/policyTags/your-policy-tag-id. You must set this as a top level parameter when you call ```dfeAnalyticsDataform()```. You may in addition set this at key level for fields to override this value - for example, if you want to set specific data masking rules on particular fields. You may not override this parameter at key level for fields that are not your primary key, or ```created_at``` or ```updated_at```. If not set then no fields will be hidden.
+- ```hidePrimaryKey``` - boolean; optional; table level. If ```true``` the primary key for this table will have the policy tag at ```hiddenPolicyTagLocation``` attached to it. The primary key is ```id``` by default, but may be configured via the table-level ```primaryKey``` parameter.
+- ```hideCreatedAt``` - boolean; optional; table level. If ```true``` the ```created_at``` field for this table will have the policy tag at ```hiddenPolicyTagLocation``` attached to it.
+- ```hideUpdatedAt``` - boolean; optional; table level. If ```true``` the ```updated_at``` field for this table will have the policy tag at ```hiddenPolicyTagLocation``` attached to it.
+- ```hidden``` - boolean; optional; key level. Set this to ```true``` for an individual key within a table to ensure that this field has a policy tag attached to it.
+
+dfe-analytics-dataform also contains two assertions which will fail to alert you when the data being streamed into the ```data``` and ```hidden_data``` fields in the ```events``` table does not match the configuration specified in the parameters above:
```

Comment:
It is that the assertions will fail or the assertions will alert you rather than fail to alert you? If this is correct, maybe the double negative with 'when ....does not match' makes it confusing

Reply:
Changing to 'fail and alert you'. The failure triggers the alert in Dataform.